README.rst

```diff
@@ -6,18 +6,27 @@ Mock services
    :target: https://circleci.com/gh/novafloss/mock-services
    :alt: We are under CI!!
 
-Aims to provide an easy way to mock an entire service API based on `responses`_
-and a simple dict definition of a service. The idea is to mock everything at
-start according the definition. Then `mock-services`_ permits to
+Aims to provide an easy way to mock an entire service API based on
+`requests-mock`_ and a simple dict definition of a service. The idea is to mock
+everything at start according given rules. `mock-services`_ permits to
```

Comment:
À chaque fois que je vois `it permits to` ça me fait bizarre. C'est vraiment très formel, en général on dit `it allows to`.